zppy/mpas_analysis.py

```diff
@@ -1,14 +1,17 @@
 import jinja2
 import os
-
-from utils import getTasks, getYears, submitScript, checkStatus
+from importlib import resources
+from zppy.utils import getTasks, getYears, submitScript, checkStatus
 
 # -----------------------------------------------------------------------------
 def mpas_analysis(config, scriptDir):
 
     # Initialize jinja2 template engine
     templateLoader = jinja2.FileSystemLoader(searchpath=config['default']['templateDir'])
     templateEnv = jinja2.Environment( loader=templateLoader )
+    # TODO: need to get path from package passed to it (importlib.resource.path)
+    #with resources.path('zppy.templates', 'mpas_analysis.bash') as target:
+    #    os.symlink(str(target), '{}/'.format('???'))
```

Comment:
@xylar I was trying to emulate https://github.com/MPAS-Dev/compass/blob/864ebf7425d5f076d4c18673b43d102308da7071/compass/landice/tests/enthalpy_benchmark/B/__init__.py#L44-L46, but it seems like `zppy -c <config-file>` works fine without `importlib.resource.path`. Is that just a quirk of how I'm testing?

Reply:
I think this is definitely a requirement.  I do guess that things are working for you because of how you are testing. Could you send me (e.g. on Slack) the procedure your're using for testing, including your config file so I can also do some testing?

Reply:
Hmm, I see now why it's working.  I still think using `importlib.resource.path` is the preferred way but I'll comment below.